### PR TITLE
fix(goals): send origin to analytics

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts [hunk 1]
```diff
@@ -99,34 +99,39 @@ const analyticsMiddleware = () => store => next => action => {
...
+      case AT.components.identityVerification.INITIALIZE_VERIFICATION:
+        analytics.push(AnalyticsKey.KYC_INITIALIZE_VERIFICATION, {
+          type: AnalyticsType.EVENT,
+          originalTimestamp: getOriginalTimestamp(),
+          language: language
+        })
+        break
+    }
+  } catch (e) {
+    // eslint-disable-next-line
```
Comment (pedroapfilho) [inline packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: `analytics` is not `async`, so you can just not have that

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: what do you mean?

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: This line was catching and causing the app to stop responding to actions:

> modalOriginDictionary(action.payload.props.origin)

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: This is actually causing multiple issues:

https://blockchain.atlassian.net/browse/FWLT-1339

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: yeah i think this makes sense. gonna merge and deploy to unblock QA

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: @pedroapfilho we can/may refactor all of this with Segment work

**Review by schnogz — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts (hunk 1 above)
Comment (pedroapfilho) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts:133]: AH, alright, my bad, I thought the `origin` was required

**Pull request merged**